config/fraudbusters-mgmt/values.yaml.gotmpl

```
@@ -108,10 +140,10 @@ ciliumPolicies:
   - filters:
     - port: 8022
       type: TCP
-    name: hellgate
+    name: fraudbusters-notificator
     namespace: {{ .Release.Namespace }}
   - filters:
-    - port: 8022
-      type: TCP
-    name: fault-detector
-    namespace: {{ .Release.Namespace }}
+      - port: 8022
+        type: TCP
+    name: fraudbusters
+    namespace: { { .Release.Namespace } }
```

Пробелы лишние затесались
```suggestion
    namespace: {{ .Release.Namespace }}
```